src/Microsoft.ML.StaticPipe/TreeTrainersStatic.cs

```diff
@@ -39,10 +39,10 @@ public static class TreeRegressionExtensions
         /// </example>
         public static Scalar<float> FastTree(this RegressionCatalog.RegressionTrainers catalog,
             Scalar<float> label, Vector<float> features, Scalar<float> weights = null,
-            int numLeaves = Defaults.NumLeaves,
-            int numTrees = Defaults.NumTrees,
-            int minDatapointsInLeaves = Defaults.MinDocumentsInLeaves,
-            double learningRate = Defaults.LearningRates,
+            int numLeaves = Defaults.NumberOfLeaves,
```

Comment:
>numLeaves [](start = 16, length = 9)

Shouldnt these parameters also be fully named?

Reply:
Thanks! I didn't realize they are FastTree static APIs.

---
In reply to: [261464899](https://github.com/dotnet/machinelearning/pull/2753#discussion_r261464899) [](ancestors = 261464899)